<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtVideo</class>
 <widget class="QMainWindow" name="QtVideo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {background-color: rgba(0,0,0,0);}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayoutCentralWidget">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="View::VideoPlayer" name="viewVideoPlayer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="0" alignment="Qt::AlignBottom">
     <widget class="QWidget" name="widgetInterfacePlayer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <layout class="QHBoxLayout" name="hBoxLayoutInterface" stretch="0,0,0,1,0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item alignment="Qt::AlignVCenter">
        <widget class="QPushButton" name="pushButtonPlayStop">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="icon">
          <iconset>
           <activeoff>:/img/player/play.png</activeoff>
           <activeon>:/img/player/pause.png</activeon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonStepDown">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <disabledon>:/img/player/stepDown.png</disabledon>
          </iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonStepUp">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <disabledon>:/img/player/stepUp.png</disabledon>
          </iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignVCenter">
        <widget class="SliderPlayer" name="sliderVideoDuration">
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignVCenter">
        <widget class="QPushButton" name="pushButtonSound">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <activeoff>:/img/player/soundOn.png</activeoff>
           <activeon>:/img/player/soundOff.png</activeon>
          </iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SliderPlayer" name="sliderSoundDuration">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="tracking">
          <bool>true</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="openFille"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="openFille">
   <property name="text">
    <string>Открыть Файл...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SliderPlayer</class>
   <extends>QSlider</extends>
   <header>sliderPlayer.h</header>
  </customwidget>
  <customwidget>
   <class>View::VideoPlayer</class>
   <extends>QWidget</extends>
   <header>view/videoPlayer/videoPlayer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButtonPlayStop</tabstop>
  <tabstop>sliderVideoDuration</tabstop>
  <tabstop>pushButtonSound</tabstop>
  <tabstop>sliderSoundDuration</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <signal>signalFullScreen(bool)</signal>
  <slot>showFullScreen(bool)</slot>
 </slots>
</ui>
